src/webhooks/registry.ts

```diff
@@ -147,9 +143,6 @@ function buildQuery(
       webhookSubscriptionArgs = `{callbackUrl: "${address}"}`;
       break;
     case DeliveryMethod.EventBridge:
```

Comment:
This case is OK, but I think the previous one could be changed:
```ts
case DeliveryMethod.Http || !versionSupportsEndpointField():
```

Since `Http` will always have a value, the expression will always evaluate to `DeliveryMethod.Http`, and the version check never runs (which is OK since we're doing the version check earlier now!).

Reply:
Good point, I'll remove that.